integration-tests/micrometer-jmx/pom.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-integration-tests-parent</artifactId>
+    <version>999-SNAPSHOT</version>
+    <relativePath>../</relativePath>
+  </parent>
+
+  <artifactId>quarkus-integration-test-micrometer-jmx</artifactId>
+  <name>Quarkus - Integration Tests - Micrometer JMX</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-micrometer</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+    <dependency>
```

Comment:
Is `micrometer-core` needed as a dependency if the micrometer extension is present?

Reply:
I removed this yesterday. ;)